docs/_source/guides/train_a_model.ipynb

```diff
@@ -0,0 +1,358 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "d09c1533-97fd-43f7-83ea-f3a56edd1d5e",
+   "metadata": {},
+   "source": [
+    "# 🦾 Train a Model\n",
+    "\n",
+    "This guide showcases how to train a model on the the `Dataset` classes in the Argilla client.\n",
+    "The Dataset classes are lightweight containers for Argilla records. These classes facilitate importing from and exporting to different formats (e.g., `pandas.DataFrame`, `datasets.Dataset`) as well as sharing and versioning Argilla datasets using the Hugging Face Hub. For more info, take a look at the\n",
+    "\n",
+    "For each record type there's a corresponding Dataset class called `DatasetFor<RecordType>`.\n",
+    "You can look up their API in the [reference section](../reference/python/python_client.rst#module-argilla.client.datasets)."
+   ]
```

Comment:
I would add a brief outline of the two main sections. 

Something like:

There are two ways to train custom models on top of your annotated data:

1. Train models using the Argilla training module.
2. Train with a custom workflow using the prepare for training methods

I would change the order of the sections, i.e., Argilla trainer should go first. Also if you feel like adding a sentence about when to use what in the outline that's great. Something like: Custom workflows give you more flexibility to integrate with your existing training workflows.